flink-connector-mysql-cdc/src/test/java/com/alibaba/ververica/cdc/connectors/mysql/source/split/MySqlSplitSerializerTest.java

```diff
@@ -96,7 +95,7 @@ public void testBinlogSplit() throws Exception {
 
         final MySqlSplit split =
                 new MySqlBinlogSplit(
-                        BINLOG_SPLIT_ID,
+                        "binlog-split",
```

Comment:
nit: use `BINLOG_SPLIT_ID`

Reply:
I just don't want to make `BINLOG_SPLIT_ID` to be public, and we should be able to use arbitrary split id here.